src/bots/grammar.py

```diff
@@ -1097,7 +1097,7 @@ class avro(Grammar):
         'I': 'I',  # numercial, implicit decimal
         'B': 'B', # Boolean
         'ENUM': 'ENUM',
-        'UUID': 'UUID'
+        'UUID': 'A'
```

Comment:
What's the meaning of this change?

Reply:
Nothing really but since bots stores all values as string a UUID internal type doesnt make any sense